tests/converter/test_group.py

```diff
@@ -206,35 +190,6 @@ def test_inner_shadows_background(self):
             InnerShadow(blurRadius=4, offsetX=1, offsetY=3, spread=0, color=SKETCH_COLOR[1])
         ]
 
-    def test_section_as_group(self):
-        fig_section = {
-            **FIG_BASE,
-            "type": "SECTION",
-            "resizeToFit": False,
-            "fillPaints": [
-                {
-                    "type": "SOLID",
-                    "color": FIG_COLOR[0],
-                    "opacity": 0.9,
-                    "visible": True,
-                }
-            ],
-            "children": [{**FIG_BASE, "type": "ROUNDED_RECTANGLE"}],
-        }
-
-        group = tree.convert_node(fig_section, "")
-
-        assert group._class == "group"
-        assert group.style.fills == []
-        assert group.style.borders == []
-
-        clip = group.layers[0]
-        assert clip.hasClippingMask
-
-        bg = group.layers[1]
-        assert len(bg.style.fills) == 1
-        assert bg.style.fills[0].fillType == FillType.COLOR
```

Comment:
Sections can now always be nested frames. No need to use groups to support nesting in Sketch